### ISSUE TITLE
D-Lite tie in normal/snail painters

### ISSUE THREAD

**Issue by IanMayo** [debrief/debrief#3682]
Once we have connected the time controller to the plot (#3681 ), we should tie in the normal/snail painters.

**Comment by IanMayo** [comment]
@saulhidalgoaular - I've made _Some_ progress on this, in this PR:  #3690

But, I've got stuck.   I know the `TotePainter` and `SnailPainter` methods are getting called when the time changes, but their graphics just don't get rendered on the screen.

Once a `.rep` file is loaded, the time controls become enabled.  In `Normal` mode, a rectangle should be drawn around the current track location.  In `Snail` mode, a course/speed sensitive vector should be drawn at the current track location.

I do get some occasional rendering, strangely as D-Lite loses focus - but I'm stumped :-)
